### DISCUSSION TITLE
Show answered discussion count on mobile

### DISCUSSION THREAD

**Discussion by RedcodesDev** [community/community#13083] [Mobile] [answered]
I would like to suggest to show the amount of answered discussions on a github profile on mobile. This is already on Desktop in the "Highlights" section of every profile and would welcome to also see it in the app.

**Comment by aconsuegra** [comment, marked as answer]
Thanks for your feature request @RedcodesDev! This is a good idea, I have created an issue on our end to track it. Keep the feedback coming!
